VRDR/DeathRecord.cs

```diff
@@ -389,13 +389,15 @@ private void CreateDeathDateObs()
             DeathDateObs.Meta.Profile = deathdate_profile;
             DeathDateObs.Status = ObservationStatus.Final;
             DeathDateObs.Code = new CodeableConcept(CodeSystems.LOINC, "81956-5", "Date+time of death", null);
-            DeathDateObs.Subject = new ResourceReference("urn:uuid:" + Decedent.Id);
+            DeathDateObs.Subject = new ResourceReference("urn:uuid:" + (Decedent!=null?Decedent.Id:"000000000"));
```

Comment:
I think the right thing to do here is to not set the Subject if there isn't a Subject present.